### PR TITLE
Add kubectl version

### PR DESCRIPTION
**Pull request by iranzo**
Define kubectl version tested

Also removes folder specific to O/S left overs

### CODE REVIEW

File: pages/quickstart_minikube.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ _(*): KubeVirt >=v0.9.2 won't run on Minikube <v0.35.0 by default because of lac
 
 Finally, you'll need *kubectl* installed, it can be downloaded from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl){:target="_blank"} or installed using the means available for your platform.
 
+_(*): Latest validated with *kubectl* version v1.12.5
+
```
Comment (codificat) [inline pages/quickstart_minikube.md]: Instead of mentioning a specific version it might be better to recommend a version check on `kubectl` that complies with the [supported release skew](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).

In other words: use a `kubectl` version that is *close enoug* to your kubernetes server version

Comment (iranzo) [reply pages/quickstart_minikube.md]: Changed in latest push

File: pages/quickstart_minikube.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ _(*): KubeVirt >=v0.9.2 won't run on Minikube <v0.35.0 by default because of lac
 
 Finally, you'll need *kubectl* installed, it can be downloaded from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl){:target="_blank"} or installed using the means available for your platform.
 
+_(*): Ensure that *kubectl* version complies with the [supported release skew](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew).
```
Comment (codificat) [inline pages/quickstart_minikube.md]: Syntax: unmatched underscore for italics.

Also the star (*) isn't present in the text.

I believe the text might be too "technical" or detailed/confusing for the quickstart guide. Maybe also mention what this means in plain english? Something along the lines of "the version of `kubectl` should match the kubernetes server version as close as possible".

Comment (iranzo) [reply pages/quickstart_minikube.md]: Thanks, added

File: pages/quickstart_minikube.md [hunk 3]
```diff
@@ -19,7 +19,9 @@ driver.
 
 _(*): KubeVirt >=v0.9.2 won't run on Minikube <v0.35.0 by default because of lack of vhost-net device. (see more details at [Issue#2056](https://github.com/kubevirt/kubevirt/issues/2056))_
 
-Finally, you'll need *kubectl* installed, it can be downloaded from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl){:target="_blank"} or installed using the means available for your platform.
+Finally, you'll need *kubectl* installed (*), it can be downloaded from [here](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl){:target="_blank"} or installed using the means available for your platform.
+
+_(*): Ensure that *kubectl* version complies with the [supported release skew](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) (version of kubectl whould be close to kubernetes server version).    _
```
Comment (codificat) [inline pages/quickstart_minikube.md]: Typo: `whould`. Maybe *should* instead while we're at it?. Also for grammar add a couple of *the*: /The version/ and /the Kubernetes/ (also capital K).

Comment (iranzo) [reply pages/quickstart_minikube.md]: thanks!

**Comment by codificat** [comment]
Ok thanks - one final request: can you squash please? There are more commits than lines changed :smiley:

**Comment by iranzo** [comment]
Done
